lang/en.yml

```diff
@@ -40,7 +40,7 @@ en:
     ERR_CONFIG_NO_ORGANISATION_URL: "No organisation URL is specified in YML configuration. Ensure the 'metadata_organisation_url' value is defined in your YML configuration"
     ERR_CONFIG_NO_SUPPORT_CONTACT: 'Support contact detail is missing from YML configuration. Ensure the following values are defined in the YML configuration: metadata_contact_support_company, metadata_contact_support_firstnames, metadata_contact_support_surname'
     ERR_ENV_NOT_ALLOWED: "The RealMe environment specified on the cli ('{env}') is not allowed. It must be one of: {allowedEnvs}"
-    ERR_ENV_NOT_SPECIFIED: 'The RealMe environment was not specified on the cli It must be one of: {allowedEnvs} e.g. vendor/bin/sake dev/tasks/RealMeSetupTask forEnv=mts'
+    ERR_ENV_NOT_SPECIFIED: 'The RealMe environment was not specified on the cli It must be one of: {allowedEnvs} e.g. vendor/bin/sake tasks:RealMeSetupTask --forEnv=mts'
```

Comment:
That probably has to be a new key. Also need to double check what's going on with those phone number changes.